Shared/Widget/WidgetDataEncoder.swift

```diff
@@ -50,7 +50,7 @@ struct WidgetDataEncoder {
 			}
 			
 			// Today Articles
-			let todayArticles = try SmartFeedsController.shared.todayFeed.fetchArticles().sorted(by: { $0.datePublished ?? .distantPast > $1.datePublished  ?? .distantPast  })
+			let todayArticles = try SmartFeedsController.shared.todayFeed.fetchArticles().sorted(by: { $0.datePublished ?? .distantPast > $1.datePublished  ?? .distantPast  }).filter({ $0.status.read == false })
 			var today = [LatestArticle]()
```

Comment:
I think you want to use `fetchUnreadArticles` here instead of manually filtering them.  Same on line 67.

Reply:
Yup!